benchmarks/test_benchmarks.py

```diff
@@ -0,0 +1,31 @@
+
+import time
+import sys
+
+import pytest
+import subprocess
+
+from benchmarks import nbody
+
+# Interpreters:
+rustpython_exe = '../target/release/rustpython'
+cpython_exe = sys.executable
+pythons = [
+    cpython_exe,
+    rustpython_exe
+]
+
+# Benchmark scripts:
+benchmarks = [
+    ['benchmarks/nbody.py'],
+    ['benchmarks/mandelbrot.py'],
+]
+
+@pytest.mark.parametrize('exe', pythons)
+@pytest.mark.parametrize('args', benchmarks)
+def test_bench(exe, args, benchmark):
```

Comment:
is this being run on every `pytest` invocation?

Reply:
No, it is not run on travis, but when you run pytest in the benchmark directory, it will run test_bench for each snippet and each interpreter.